web/app/__init__.py

```diff
@@ -31,6 +32,7 @@ def get_app(config):
     app.logger.addHandler(stream_handler)
 
     db.init_app(app)
+    Migrate(app, db)
```

Comment:
What does this Migrate function do? Every time the app starts up it tries to do a database migration?